src/visibility.js

```diff
@@ -155,7 +156,29 @@ class Visibility extends BasePlugin {
     }
   }
 
+  _stopFloatingOnOtherPlayers() {
```

Comment:
the method duplicates method `_stopFloating`
I would recommend add to `_stopFloating` method params floatingPoster, floatingContainer (by default keep `this._floatingPoster` and `this._floatingContainer`) then use `_stopFloating` method with proper params inside `_stopFloatingOnOtherPlayers`